charts/fluent-operator/templates/fluentbit-output-elasticsearch.yaml

```diff
@@ -34,7 +34,12 @@ spec:
 {{ toYaml .Values.fluentbit.output.es.httpPassword | indent 6 }}
 {{- end }}
     logstashFormat: {{ .Values.fluentbit.output.es.logstashFormat | default true }}
+{{- if .Values.fluentbit.output.es.logstashPrefix }}
     logstashPrefix: {{ .Values.fluentbit.output.es.logstashPrefix | default "ks-logstash-log" | quote }}
+{{- end }}
+{{- if .Values.fluentbit.output.es.logstashPrefixKey }}
+    logstashPrefixKey: {{ .Values.fluentbit.output.es.logstashPrefixKey | default "ks-logstash-log-key" | quote }}
```

Comment:
https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator/values.yaml#L220
The handling here needs to be similar to that of `logstashPrefix`, which you can add and comment out in `value.yaml`.
Also, if you're using `if` for judgement, then there's no need to use `default`.

Reply:
thanks for review, I have been modify.